src/test/java/com/commercetools/sync/internals/helpers/CustomHeaderSphereClientDecoratorTest.java

```diff
@@ -0,0 +1,40 @@
+package com.commercetools.sync.internals.helpers;
+
+import io.sphere.sdk.client.HttpRequestIntent;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.client.SphereRequest;
+import io.sphere.sdk.http.HttpHeaders;
+import io.sphere.sdk.models.Versioned;
+import io.sphere.sdk.producttypes.ProductType;
+import io.sphere.sdk.producttypes.commands.ProductTypeUpdateCommand;
+import java.util.Collections;
+
+import io.sphere.sdk.producttypes.commands.updateactions.ChangeName;
+import org.junit.jupiter.api.Test;
+
+import static com.commercetools.sync.commons.utils.SyncSolutionInfo.LIB_VERSION;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+
+public class CustomHeaderSphereClientDecoratorTest {
+
+    private static final String LIB_NAME = "commercetools-sync-java";
```

Comment:
you might use the SyncSolutionInfo.LIB_NAME

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/636/commits/843a09c90ff708ec0ad4c6a6ecf3de92a8e62cbb